mscore/inspector/inspector_stafftypechange.ui

```diff
@@ -114,7 +114,7 @@
       <item row="11" column="0" colspan="2">
        <widget class="QCheckBox" name="slashStyle">
```

Comment:
Don't we want to rename the widget as well? Otherwise, the combination of names looks really weird :)

Reply:
OK, guess we could...

Reply:
But maybe better not, `slashStyle` is used in many many places in the source, apparently for hysterical raisins ;-) See also @MarcSabatella's remark about that in the issue

Reply:
It is indeed the same setting used in Staff Properties, where the widgets themselves are called stemlessPitched and stemlessPercussion but the corresponding member of StaffType class is called slashStyle.  For tab, it's set as a side effect of the radio button to control stems.  I've never understood why this name existed, but guessed it either had something to do with tab or else it predated our support for ther forms of slash notation, and setting a staff to stemless was the first step in creating slash notation.  Maybe even at one time this setting also forced the notes to slash heads.  We'd probably have to go back to sourceforge or pick @wschweer or @lasconic memory to know.

Anyhow, I'm fine with a global replace of slashStyle to "stemless", except for compatibility we probably shouldn't change the tag written to the MSCX (or we should write both forms always and accept either on read).

Reply:
That'd be a rather big change... for IMHO very little benefit

Reply:
Agreed.  I said I'm fine with it, but I don't actually recommend it.

Reply:
I'll have the changes for this ready shortly (but no idea what might break along the way), will make it a separate commit, for the decision takers to see ;-)